spec/feature_flagger/feature_spec.rb

```diff
@@ -1,12 +1,14 @@
 require 'spec_helper'
+require 'feature_flagger/manifest_sources/with_yaml_file'
 
 module FeatureFlagger
   RSpec.describe Feature do
     subject { Feature.new(key, :feature_flagger_dummy_class) }
 
     before do
-      filepath = File.expand_path('../../fixtures/rollout_example.yml', __FILE__)
-      FeatureFlagger.config.yaml_filepath = filepath
+      yaml_path = File.expand_path('../../fixtures/rollout_example.yml', __FILE__)
+      allow(FeatureFlagger.config).to receive(:manifest_source).and_return(yaml_path)
+      allow(FeatureFlagger.config).to receive(:info).and_return(YAML.load_file(yaml_path))
```

Comment:
```suggestion
  FeatureFlagger.config.manifest_source = FeatureFlagger::ManifestSources::WithYamlFile.new(yaml_path)
```

be careful with mocks.